src/IndexingWrapper.cpp

```diff
@@ -98,13 +98,13 @@ parseField(const string & field, const char & separator)
 //' @param dbpath	path to a Xapian database
 //' @param dataFrame	data frame to be indexed
 //' @param idColumn	index of a column in the data frame whose row value
-//will be used as a unique identifier to each document(row)
+//                  will be used as a unique identifier to each document(row)
```

Comment:
Yes. It's needed at the beginning of each line.

On Tue, Aug 16, 2016 at 7:17 PM, James Aylett <notifications@github.com>
wrote:

> In src/IndexingWrapper.cpp
> <https://github.com/amandaJayanetti/RXapian/pull/28#discussion_r74937179>:
>
> > @@ -98,13 +98,13 @@ parseField(const string & field, const char & separator)
> >  //' @param dbpath	path to a Xapian database
> >  //' @param dataFrame	data frame to be indexed
> >  //' @param idColumn	index of a column in the data frame whose row value
> > -//will be used as a unique identifier to each document(row)
> > +//                  will be used as a unique identifier to each document(row)
>
> This line doesn't align, I suspect because when you're editing it you have
> different tab settings to the usual. Maybe get rid of the tabs in the rest
> of this comment block? (eg @param idColumn above is followed by a tab.)
>
> Is the //' only needed at the start of each @param line? I assume it's
> something to do with Roxygen or similar.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amandaJayanetti/RXapian/pull/28/files/5133cfbf4a41265c70673f0524ca45f155fc90d0..cb84655f77364a7ea548ae930207b02a8caf5303#r74937179>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH_jNpNG-EA-tc1VVXSvPVMzcxLARHwXks5qgb9ugaJpZM4Jj8cQ>
> .
>